cds/modules/records/templates/cds_records/record_detail.html

```diff
@@ -93,11 +93,13 @@
     {% set user_action_media_download_url = '' %}
     {% endif %}
     {# TODO: remove and replace with jinja ? #}
+    {% set related_query_url = config.CDS_RECORDS_RELATED_QUERY or '' %}
```

Comment:
this one should change to a different config right? Maybe we have something per related_identifier scheme?

```
CDS_SEARCH_RELATED_IDENTIFIERS_BY_SCHEME = {
  "indico": {
      "landing_page_section_label": "More videos from this Indico event",
      "query": "...."
  }
}
```

Reply:
You are right :) Let's leave it like that for the moment and see if we have more cases e.g other schemes to group on then we can think of a new config